Cargo.lock

```diff
@@ -581,17 +594,42 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "ec519da181d4ea9a3184a8a4ec9ed597a6f080c4505717a5fca174e739aff2ec"
 dependencies = [
  "anyhow",
- "cairo-lang-defs",
- "cairo-lang-diagnostics",
- "cairo-lang-filesystem",
- "cairo-lang-lowering",
- "cairo-lang-parser",
- "cairo-lang-project",
- "cairo-lang-semantic",
- "cairo-lang-sierra",
- "cairo-lang-sierra-generator",
- "cairo-lang-syntax",
- "cairo-lang-utils",
+ "cairo-lang-defs 2.9.1 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
you're now shipping two copies of the compiler that are potentially incompatible, that's bad

I'm working on a proper solution for this but it will take me some more hours to ship